### ISSUE TITLE
Client-side application error on refresh with drawing mark

### ISSUE THREAD

**Issue by mcbouslog** [zooniverse/front-end-monorepo#3251]
## Package
- lib-classifier?

## Describe the bug
Create a transcription line with subtask text, refresh the page (will prompt "Reload?", click "Reload"), then client-side application error.
To clear error must clear browser session storage.

Bug - https://www.zooniverse.org/projects/humphrydavy/davy-notebooks-project?demo=true
Similar project without bug - https://www.zooniverse.org/projects/rachaelsking/corresponding-with-quakers?demo=true

## To Reproduce
1. Go to https://www.zooniverse.org/projects/humphrydavy/davy-notebooks-project/classify/workflow/21240?demo=true
2. Create line, in subtask add text, click "Save and Close"
3. Refresh page (will be prompted "Reload?", click "Reload")

## Expected behavior
Do same steps as noted above on https://www.zooniverse.org/projects/rachaelsking/corresponding-with-quakers/classify/workflow/20713?demo=true - no error

**Comment by mcbouslog** [comment]
Please change this issue title once we have a better idea of it's cause.

**Comment by mcbouslog** [comment]
I think I've isolated the issue to https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/ClassifierContainer.js#L77 - 
```
const classifierStore = useHydratedStore(storeEnvironment, cachePanoptesData, `fem-classifier-${project.id}`)
```

**Comment by mcbouslog** [comment]
Introduced in #2863 (checked commit before confirmed no error, commit of PR confirmed error).

**Comment by eatyourgreens** [comment]
This should be fixed by #3293, which adds error handling to the `useHydratedStore` hook.
